test/testdata/perfsprint.go

```diff
@@ -0,0 +1,8 @@
+//golangcitest:args -Eperfsprint
+package testdata
+
+import "fmt"
+
+func SprintfCouldBeStrconv() {
+	fmt.Sprintf("%d", 42) // want "Sprintf can be replaced with faster strconv.Itoa"
```

Comment:
1. Why do not copy-paste tests from analyzer?
2. You do not test it locally (see obvious typo)

Reply:
the tests inside the linter don't use the standard testing framework for analyzer, so they cannot be copy-paste.

Reply:
If we use [the test file](https://github.com/catenacyber/perfsprint/blob/main/analyzer/testdata/src/p/p.go), there are too many reports.
Our test system is not able to handle that.